external/functional-test/test.properties

```diff
@@ -2,4 +2,6 @@ github_url="https://github.com/adoptium/aqa-tests.git"
 test_results="testResults"
 gradle_version="5.1"
 environment_variable="MODE=java CC=gcc-7 CXX=g++-7"
-ubuntu_packages="ant-contrib build-essential git asciidoc"
\ No newline at end of file
+generic_packages="git"
+ubuntu_packages="ant-contrib build-essential asciidoc"
+fedora_packages="ant-contrib asciidoc"
```

Comment:
I'm confused what is the point of change property of ubuntu_packages to generic_package and ubuntu_packages? And what is the reason that some tests add fedora_packages and some not? I had thought if we were going to split the issue into supporting global image and discussion of enhancement of generic_package all test.properties would leave as it. Did I miss something?

Reply:
In addition, the change from `ubi_packages` to `(fedora|ubi|rhel|centos)_packages` didn't work, see https://github.com/adoptium/aqa-tests/pull/5553#issuecomment-2359934873 which is already using SHA `[cecf323](https://github.com/adoptium/aqa-tests/pull/5553/commits/cecf32367af67f04556ea093ee4f0485d7913b54)` that contains `perl make` but `make` was not picked up.
Tested again, no packages picked up with current commit ` [exec] STEP 4/24: RUN dnf install -y    	&& dnf clean all` (hyc_grinder 43553). 
Do we currently have a case using `rhel`, `centos`, or `fedora` as the container OS? AFAIK, `ubi` is the container OS for `rhel`, so our customer only uses `ubi` as their supported container OS in their application. If not, I think we can keep only `ubi` for now.

Reply:
Fedora do not have `build-essential` so the install would fail.
On contrary `git` is of same name everywhere, so it to not have sense to type it twice. Does it make more sense?

Reply:
Per request I had removed usage of completely shared gnereic deps (https://github.com/adoptium/aqa-tests/pull/5553/commits/82b1c78fd18125db12037da1d4f55270cc62d717), which were declared once, and used everywhere (in all external tests). So now also the tar, wget and few more, which are copypasted in most of the test.properties, remains copypasted

Reply:
> In addition, the change from `ubi_packages` to `(fedora|ubi|rhel|centos)_packages` didn't work, see [#5553 (comment)](https://github.com/adoptium/aqa-tests/pull/5553#issuecomment-2359934873) which is already using SHA `[cecf323](https://github.com/adoptium/aqa-tests/pull/5553/commits/cecf32367af67f04556ea093ee4f0485d7913b54)` that contains `perl make` but `make` was not picked up. Tested again, no packages picked up with current commit ` [exec] STEP 4/24: RUN dnf install -y && dnf clean all` (hyc_grinder 43553). Do we currently have a case using `rhel`, `centos`, or `fedora` as the container OS? AFAIK, `ubi` is the container OS for `rhel`, so our customer only uses `ubi` as their supported container OS in their application. If not, I think we can keep only `ubi` for now.

Yes, I'm trying to run them all.  I will verygladly  elaborate on  the ubi not picked by you. Can you please share `commandline` how to hit it?

Reply:
> > In addition, the change from `ubi_packages` to `(fedora|ubi|rhel|centos)_packages` didn't work, see [#5553 (comment)](https://github.com/adoptium/aqa-tests/pull/5553#issuecomment-2359934873) which is already using SHA `[cecf323](https://github.com/adoptium/aqa-tests/pull/5553/commits/cecf32367af67f04556ea093ee4f0485d7913b54)` that contains `perl make` but `make` was not picked up. Tested again, no packages picked up with current commit ` [exec] STEP 4/24: RUN dnf install -y && dnf clean all` (hyc_grinder 43553). Do we currently have a case using `rhel`, `centos`, or `fedora` as the container OS? AFAIK, `ubi` is the container OS for `rhel`, so our customer only uses `ubi` as their supported container OS in their application. If not, I think we can keep only `ubi` for now.
> 
> 

[LongyuZhang](https://github.com/LongyuZhang) Yes, I'm trying to run them all. I will verygladly elaborate on the ubi not picked by you. Can you please share `commandline` how to hit it?

Reply:
#[LongyuZhang](https://github.com/LongyuZhang)  I cannot acces syour logs, sorry:( I tried with crystall ball, and on "my" ubi images, and it worked fine. So we have to have different setup. Please provide mas much as you can. I will really galdly fix it. This PR is of most uttermost importance for me.

Reply:
@LongyuZhang  ^^ (sorry, I messed up need-info character)